packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md

```diff
@@ -145,6 +145,24 @@ Generated by [AVA](https://avajs.dev).
       ],
     ]
 
+## upgrade; update noble ICA
+
+> Under "published", the "fastUsdc.nobleICA" node is delegated to Fast USDC.
+> The example below illustrates the schema of the data published there.
+> 
+> See also board marshalling conventions (_to appear_).
+
+    [
+      [
+        'published.fastUsdc.nobleICA',
```

Comment:
why create this node?

Reply:
Aside from being generally useful for grabbing the address for use in explorers and such, it's a handy way to tell whether this core eval succeeded.

Perhaps we should publish just the `.value` of the address?

Reply:
why not put it on `published.fastUsdc` with the other addresses? (e.g. `poolAddress`)

Reply:
sure; that will work.